plugin/plugin.go

```diff
@@ -48,6 +45,11 @@ type Args struct {
 	UseProxy                bool   `envconfig:"PLUGIN_USE_PROXY"`
 	Version                 string `envconfig:"PLUGIN_VERSION"`
 	Level                   string `envconfig:"PLUGIN_LEVEL"`
+
+	// Resubmit-specific
+	AnalysisName          string `envconfig:"PLUGIN_ANALYSIS_NAME"`
+	MaximumDuration       int    `envconfig:"PLUGIN_MAXIMUM_DURATION" default:"3"`
+	FailBuildAsScanFailed bool   `envconfig:"PLUGIN_FAIL_BUILD_AS_SCAN_FAILED" default:"false"`
 }
 
 func ValidateInputs(args Args) error {
```

Comment:
just confirm if any other input needs validation other than the three 

also set the defaults for UploadIncludesPattern and UploadExcludesPattern in Args struct like how you already added it for some fields.